packages/openapi-spec-builder/src/openapi-spec-builder.ts

```diff
@@ -112,6 +112,18 @@ export class OpenApiSpecBuilder extends BuilderBase<OpenApiSpec> {
 
     return this.withOperation(verb, path, spec);
   }
+
+  withOperationReturningStringWithStatusCode(
```

Comment:
Instead of adding more functions can we just add `statusCode` as an optional parameter for the existing functions?